Comanche055/MEASUREMENT_INCORPORATION.agc

```diff
@@ -87,29 +86,29 @@ Z123		VLOAD	MXV*
 		VLOAD
 			BVECTOR +12D	# BVECTOR (2)
 		MXV*	VAD*
-			W 	+162D,1
-			ZI 	+18D,2	# B(0)*W+B(1)*(W+54)+B(2)*(W+108) FIRST PASS
-		STORE	ZI 	+18D,2	# ZI THEN Z2 THEN Z3
+			W +162D,1
+			ZI +18D,2	# B(0)*W+B(1)*(W+54)+B(2)*(W+108)FIRST PASS
+		STORE	ZI +18D,2	# ZI THEN Z2 THEN Z3
 		TIX,1
 			INCOR1
 INCOR1		TIX,2	BON
-			Z123		# LOOP FOR Z1,Z2,Z3
+			Z123		# LOOP FOR ZI,Z2,Z3
```

Comment:
<img width="562" alt="screen shot 2018-11-12 at 17 58 06" src="https://user-images.githubusercontent.com/2200898/48337104-5ed7e680-e6a5-11e8-98b1-c0463b42459b.png">

It looks like it should be Z1 here.

Reply:
@wopian looks like a 1 to me

Reply:
Yeah, I forgot about that one. 😅